xml/art_sle_ha_pmremote.xml

```diff
@@ -274,50 +301,213 @@
    </itemizedlist>
   </sect1>
 
- <sect1 xml:id="sec.ha.pmremote.install">
-  <title>Setting Up a Cluster with Virtual Guest Nodes</title>
-  <para>In the following example setup, KVM is used for setting up the virtual
-   guest nodes:</para>
+ <sect1 xml:id="sec.ha.pmremote.install.phys-remote-nodes">
+  <title>Setting Up a Cluster with Physical Remote Nodes</title>
+  <para>
+   Before you proceed, install and set up a basic two-node cluster. This task
+   is described in <citetitle>&instquick;</citetitle>. This will lead to a
+   two-node cluster with the two physical hosts <systemitem
+    class="domainname">&node1;</systemitem> and <systemitem
+     class="domainname">&node2;</systemitem>.
+  </para>
+  <para>
+   In the following example setup, a physical remote node
+   <systemitem class="domainname">&node3;</systemitem> is used:
+  </para>
   <orderedlist>
    <listitem>
-    <para><xref linkend="sec.ha.pmremote.config.phys.host"/></para>
+    <para><xref linkend="sec.ha.pmremote.config.uc1.virt-guest-nodes"/></para>
    </listitem>
    <listitem>
-    <para><xref linkend="sec.ha.pmremote.config.kvm.guest"/></para>
+    <para><xref linkend="sec.ha.pmremote.config.uc1.kvm.guest"/></para>
    </listitem>
    <listitem>
-    <para><xref linkend="sec.ha.pmremote.integrate.guestsintocluster"/></para>
+    <para><xref linkend="sec.ha.pmremote.config.uc1.integrate"/></para>
    </listitem>
   </orderedlist>
 
-  <note>
-   <title>Use Static Host Names and IP Addresses</title>
-   <para>Add static hosts and IP addresses in <filename>/etc/hosts</filename>.
-   </para>
-  </note>
-
-  <sect2 xml:id="sec.ha.pmremote.config.phys.host">
+  <sect2 xml:id="sec.ha.pmremote.config.uc1.virt-guest-nodes">
    <title>Setting up a Corosync Cluster with Physical Hosts</title>
```

Comment:
'Configuring Corosync Cluster Nodes' is very general. What we show here is more 'enhance cluster nodes by creating authkey to support pacemaker_remote'. Wouldn't it be better to add some more specific title, like 'Configuring Authentication Key'?